core/src/main/java/me/aurium/branch/fallback/permissions/EmptyPermission.java

```diff
@@ -0,0 +1,39 @@
+/*
+ *
+ *  Branch
+ *  Copyright © 2021 Aurium
+ *
+ *  Branch is free software: you can redistribute it and/or modify
+ *  It under the terms of the GNU Affero General Public License as
+ *  published by the Free Software Foundation, either version 3 of the
+ *  License, or (at your option) any later version.
+ *
+ *  Branch is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ *  GNU Affero General Public License for more details.
+ *
+ *  You should have received a copy of the GNU Affero General Public License
+ *  along with Branch. If not, see <https://www.gnu.org/licenses/>
+ *  and navigate to version 3 of the GNU Affero General Public License.
+ *
+ */
+
+package me.aurium.branch.fallback.permissions;
+
+/**
+ * Represents a permission that is always true
+ * @param <T> sender
+ */
+public class EmptyPermission<T> implements Permission<T> {
+
+    @Override
+    public boolean attempt(T sender, String alias, String[] args) {
+        return true;
+    }
+
+    @Override
+    public String easyName() {
+        return "none"; //TODO perhaps throw? This shouldn't happen.
```

Comment:
throw if anything which shouldn't happen happens

Reply:
the EmptyPermission and EmptyBlock both need reworking on logic